.github/workflows/tpu-tests.yml

```diff
@@ -54,7 +54,7 @@ jobs:
 
           ## Install openblas, mkl, gsutil
           sudo apt-get install -y libopenblas-dev libomp5-10 libomp5
-          pip install mkl requests gsutil
+          pip install mkl==2021.4.0 requests gsutil
```

Comment:
@fco-dv could you please add a comment about why we fix the version, e.g. put the ref to our issue.